### PR TITLE
Lily/CommentBlocks: Add coveted C block

### PR DESCRIPTION
**Pull request by LilyMakesThings**
Soon

### CODE REVIEW

File: extensions/Lily/CommentBlocks.js [hunk 1]
```diff
@@ -4,88 +4,105 @@
...
     }
 
-    commentBoolean(args) {
+    commentBoolean (args) {
       return args.INPUT || false;
     }
   }
   Scratch.extensions.register(new CommentBlocks());
+})(Scratch);
+  Scratch.extensions.register(new CommentBlocks());
```
Comment (NexusKitten) [inline extensions/Lily/CommentBlocks.js:107]: The extension is defined twice, is that intentional?

Comment (LilyMakesThings) [reply extensions/Lily/CommentBlocks.js]: who doesn't want twice as many non-functioning blocks?

**Comment by horizontal-shading** [comment]
I thought you weren't allowed to make C-blocks

**Comment by LilyMakesThings** [comment]
> I thought you weren't allowed to make C-blocks

Soon

**Pull request merged**